README.md

```diff
@@ -146,197 +148,6 @@ iptables::rule { 'example':
 
 ## Reference
 
-<<<<<<< be8c418e96cf122b1636397d7921baf0d5b33b86
-### Native Types
-
-#### `iptables_rule`
-
-A managed iptables rule. This may be used to provide ultimate flexibility in managing the system rules.
-
-##### Parameters
-
-* name: A unique name for the rule, does not provide any other function. Required
-* comment: A comment to apply to the rule. Default: No Comment
-* header: Whether or not to auto-include the table LOCAL-INPUT in the rule. Default: true
-* apply_to: What version(s) of iptables to which to apply this rule. Default: 'auto'
-  If set to 'auto' (the default) then we'll try to guess what you want and default to ['ipv4','ipv6'].
-  If 'auto' is set then each line will be evaluated as an independent rule.
-    * Any rules that have IPv4 addresses will be applied to iptables.
-    * Any rules that have IPv6 addresses will be applied to ip6tables.
-    * All other rules will be applied to *both* utilities.
-    * If in doubt, split your rules and specify your tables!
-
-* table: The name of the table to which you are adding the rule. Default: 'filter'
-* first: If `true`, prepend the rule to the rule set. Default: `false`
-* absolute: If `true`, ensure that the rule is at the absolute beginning or end of the rule set. If multiple rules are marked as absolute then they are sorted by resource name. Default: `false`
-* order: The order in which the rule should appear. 1 is the minimum and 999 is the maximum. Default: '11'
-* resolve: If `true`, Puppet will resolve any hostnames in the rules prior to application to prevent iptables from starting should a name change in the future. Default: `true`
-* content: The full content of the rule that should be added to the rule set. You may place *anything* here. Required
-
-#### iptables_optimize
-#### ip6tables_optimize
```

Comment:
This is a good place for the 30,000 ft overview of the module. Technical documentation can still be looked for in the generated docs.

Reply:
Please leave this section in